creator-node/src/TranscodingQueue.js

```diff
@@ -92,18 +96,31 @@ class TranscodingQueue {
     logger.info(`Transcoding Queue: ${message} || active: ${active}, waiting: ${waiting}, failed ${failed}, delayed: ${delayed}, completed: ${completed} `)
   }
 
+  /**
+   * Logs an error status message and includes current queue info
+   * @param {object} logContext to create a logger.child(logContext) from
+   * @param {string} message
+   */
+  async logError (logContext, message) {
+    const logger = genericLogger.child(logContext)
+    const { waiting, active, completed, failed, delayed } = await this.queue.getJobCounts()
+    logger.error(`Transcoding error: ${message} || active: ${active}, waiting: ${waiting}, failed ${failed}, delayed: ${delayed}, completed: ${completed} `)
+  }
+
   /**
    * Adds a task to the queue that segments up an audio file
    * @param {string} fileDir
    * @param {string} fileName
    * @param {object} logContext to create a logger.child(logContext) from
    */
   async segment (fileDir, fileName, { logContext }) {
+    this.logStatus(logContext, `Adding job to segment queue, fileDir=${fileDir}, fileName=${fileName}`)
     const job = await this.queue.add(
       PROCESS_NAMES.segment,
       { fileDir, fileName, logContext }
     )
     const result = await job.finished()
+    this.logStatus(logContext, `Job added to segment queue, fileDir=${fileDir}, fileName=${fileName}`)
```

Comment:
nit - we can probably log this as `Segment job successful, fileDir=${fileDir}, fileName=${fileName}` since if line 123 is reached, that the job was successful

Reply:
also maybe add a log before 122 here to indicate that the job was successfully added to the queue

Reply:
Oh! Good catch. I thought it was *only* adding the job to the queue here. kk, ty!

Reply:
How does this and transcode320 look now?

Reply:
looks good :D